README.md

```diff
@@ -1,46 +1,110 @@
-# Getting Started with Create React App
+<div align="center">
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+<img alt="Gimesto Quiz logo" src="src/assets/favicon/favicon-32x32.png" />
+<h1>Gimesto Keep</h1>
+<br>
 
-## Available Scripts
+![Forks](https://img.shields.io/github/forks/vrx29/gimesto-keep)
+![Stars](https://img.shields.io/github/stars/vrx29/gimesto-keep)
 
-In the project directory, you can run:
+</div>
+ 
+ ---
 
-### `npm start`
+Gimesto keep is a note taking app. In this you can store your notes and edit them. There is an option to archive your notes and unarchive as well. If you delete a note it will be moved to trash/deleted page from ther you can either restore your notes or delete them permanently.
 
```

Comment:
Typo :   page from **there** you can either restore

Reply:
fixed